lizmap/modules/lizmap/lib/Server/Server.php

```diff
@@ -60,6 +64,10 @@ public function getLizmapPluginServerVersion()
      */
     public function getQgisServerVersion()
     {
+        if (array_key_exists('error', $this->metadata['qgis_server_info'])) {
+            return '0.0.0';
```

Comment:
Sorry, same question, how did you trigger this error ?

If there isn't any `qgis_server_info`, the user must see this screenshot https://github.com/3liz/lizmap-web-client/pull/3496 (the second one). If it's not the case, this would be the proper fix.

0.0.0 is not an existing version and it's mainly misleading what is displayed. Is it not generating a link to https://github.com/3liz/qgis-lizmap-server-plugin/releases/tag/0.0.0 ?

*Same question for the other 0.0.0 above*

Reply:
It is to avoid error message `Trying to access array offset on value of type null    /var/www/lizmap-web-client-3.6.2/lizmap/modules/lizmap/lib/Server/Server.php    54` in logs reported in https://github.com/3liz/lizmap-web-client/issues/3544

I return a aberrant version because it is used in `versionCompare`. I can return `Null`